crates/core/src/accessibility/tree.rs

```diff
@@ -280,12 +295,133 @@ impl AccessibilityTree {
+        }
+
+        // Rotation transform
+        if let Some((_, rotation)) = transform_state
+            .rotations
+            .iter()
+            .find(|(id, _)| id == &node_ref.id())
+        {
+            builder.set_transform(Affine::rotate(rotation.to_radians() as _));
+        }
+
+        // Clipping overflow
+        if style_state.overflow == OverflowMode::Clip {
+            builder.set_clips_children();
+        }
+
+        // Foreground/Background color
+        builder.set_foreground_color(skia_color_to_rgba_u32(font_style_state.color));
+        if let Fill::Color(color) = style_state.background {
+            builder.set_background_color(skia_color_to_rgba_u32(color));
+        }
+
+        // If the node is a block-level element in the layout, indicate that it will cause a linebreak.
+        if !node_type.is_text() {
+            if let NodeType::Element(node) = &*node_type {
+                // This should be impossible currently but i'm checking for it just in case.
+                // In the future, inline text spans should have their own own accessibility node,
+                // but that's not a concern yet.
+                if node.tag != TagName::Text {
+                    builder.set_is_line_breaking_object();
+                }
+            }
+        }
+
+        // Font size
+        builder.set_font_size(font_style_state.font_size as _);
+
+        // If the font family has changed since the parent node, then we inform accesskit of this change.
+        if let Some(parent_node) = node_ref.parent() {
+            if parent_node.get::<FontStyleState>().unwrap().font_family
+                != font_style_state.font_family
+            {
+                builder.set_font_family(font_style_state.font_family.join(", "));
+            }
+        } else {
+            // Element has no parent elements, so we set the initial font style.
+            builder.set_font_family(font_style_state.font_family.join(", "));
+        }
+
+        // Set bold flag for weights above 700
+        if font_style_state.font_weight > 700.into() {
+            builder.set_bold();
+        }
+
+        // Text alignment
+        builder.set_text_align(match font_style_state.text_align {
+            TextAlign::Center => accesskit::TextAlign::Center,
+            TextAlign::Justify => accesskit::TextAlign::Justify,
+            // TODO: change representation of `Start` and `End` once RTL text/writing modes are supported.
+            TextAlign::Left | TextAlign::Start => accesskit::TextAlign::Left,
+            TextAlign::Right | TextAlign::End => accesskit::TextAlign::Right,
+        });
+
+        // TODO: Adjust this once text direction support other than RTL is properly added
+        builder.set_text_direction(TextDirection::LeftToRight);
+
+        // Set italic property for italic/oblique font slants
+        match font_style_state.font_slant as u32 {
+            // Italic | Oblique
+            1 | 2 => builder.set_italic(),
```

Comment:
why are you using numbers instead of enum variants?

Reply:
Sorry, some of this code was brought over from #841 (which is now dead). I think at the time I couldn't get `Slant` in scope. I'll get that changed.